### ISSUE TITLE
Same name of app already exists + not able to fix growing/shrinking delays by selecting individual subs

### ISSUE THREAD

**Issue by jaques-sam** [cristianmiranda/subfix#1]
Hello, I stumbled upon your repo here because I was searching for a new version of SubFix that is capable of running in x64 for MacOS. The app I'm talking about can be found here https://alternativeto.net/software/subfix/.
I make here an issue, not only because you used the same name of a much better GUI tool. The tool is the best subtitle tool I have ever used. Just because it uses a different mechanism instead of adding/removing a delay. Sometimes the delay can grow/shrink over time. The tool is so simple, giving you the capability of just setting the correct time for 2 separate subtitles in the beginning and end of the movie. Subfix will interpolate for all the other subtitles in the file.

I hoped to see this in your tool too :-)

**Comment by cristianmiranda** [comment]
Hi! The goal of this tool is just to be able to fix subtitles from command line (which the GUI tool doesn't allow afaik). I don't care about naming too much, mostly because this is a command line tool and it is based on some other tool called subsync (not related to macOS subfix at all).

**Comment by jaques-sam** [comment]
Yes I understand. Maybe I didn't explain it well enough... What the GUI SubFix tool is so good at, is that you can select a subtitle in the beginning, set the correct time, and do the same for one at the end. This will automatically adjust all subs. So this is almost what your tool is able to do with e.g. `subfix @+1 @+20`, only you have to calculate the timings yourself... If you have an idea how to handle this, I will help. This would make this command line tool perfect for most people ;-)
